task-json/fetch.js

```diff
@@ -0,0 +1,22 @@
+global.fetch = require("cross-fetch");
+
+
+function readJson () {
+    fetch('https://gist.githubusercontent.com/keeguon/2310008/raw/bdc2ce1c1e3f28f9cab5b4393c7549f38361be4e/countries.json')
+    .then(response => {
+        if (!response.ok) {
+            console.error("HTTP error " + response.status);
+        }
+        const body =  response.text();
+        return body;

```

Comment:
- there is no need to create a variable if it will not be processed
- return the value as json not text

Reply:
when i return a response.json() it getting me an error   type: 'invalid-json' and invalid token at position 6

Reply:
the fetch returns data type called `Response`
you can read more about it in this link
https://developer.mozilla.org/en-US/docs/Web/API/Response

Reply:
i have read it and i found out i think there is a problem in the json array  maybe a space or something that why it wouldn't convert it as json and giving me invalid type is that possible ?